### PR TITLE
[doc][yba] Add same version requirement for repl -> yba installer migration

### PR DESCRIPTION
**Pull request by iSignal**

### CODE REVIEW

**Comment by yushenng** [comment]
I'd revise the text:

FROM: 
2. It is recommended that you migrate to the same version of YBA Installer as the original Replicated installation. If needed, perform a Replicated upgrade first to get to an equivalent version where YBA Installer.

TO: 
2. Migrations from Replicated to YBA Installer are supported in the same version family (e.g. 2.20.x to 2.20.y).  If your current YBA is at 2.16.x or earlier, you must first upgrade (using Replicated) to a version family where YBA Installer is supported (i.e. 2.18.x, 2.20.x, or later).   Then subsequently perform the migration.

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md [hunk 1]
```diff
@@ -231,18 +231,19 @@ YugabyteDB Anywhere will end support for Replicated installation at the end of 2
 
 If your YBA installation uses Replicated, you can use YBA Installer to migrate from Replicated.
 
-Review the [prerequisites](../../prerequisites/installer/). YBA Installer performs the migration in place. Make sure you have enough disk space on your current machine.
+### Before you begin
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md:234]: @ddhodge - Maybe you can copy these steps to your Prerequisites reorganization PR - https://github.com/yugabyte/yugabyte-db/pull/21876/files

Comment (iSignal) [reply docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md:234]: @ddhodge @aishwarya24 : That works, let me know if I can abandon this PR

**Comment by ddhodge** [comment]
Change also made in new PR

**Pull request merged**